src/core/server/http/integration_tests/http_service.test.ts

```diff
@@ -109,15 +111,12 @@ describe('http service', () => {
     });
 
     describe('#registerOnRequest()', () => {
-      const dummyOnRequestPlugin = path.resolve(
-        __dirname,
-        './__fixtures__/plugins/dummy_on_request'
-      );
+      const plugin = path.resolve(__dirname, './__fixtures__/plugins/dummy_on_request');
```

Comment:
TODO for myself: create an issue to cleanup tests. after https://github.com/elastic/kibana/pull/35297 we can use `setup`/ `start` contracts in tests and don't need to declare a plugin for this

Reply:
created issue https://github.com/elastic/kibana/issues/36256